builders/integrations/oracles/api3.md

```diff
@@ -222,221 +222,14 @@ The example contract contains two functions:
-
-    |                  Variable                   |                             Value                              |
-    |:-------------------------------------------:|:--------------------------------------------------------------:|
-    |          ANU QRNG Airnode Address           |         `{{ networks.moonriver.api3.anuqrngairnode }}`         |
-    |            ANU QRNG Airnode xpub            |          `{{ networks.moonriver.api3.anuqrngxpub }}`           |
-    |          ANU Endpoint ID (uint256)          |         `{{ networks.moonriver.api3.anuqrnguint256 }}`         |
-    |         ANU Endpoint ID (uint256[])         |      `{{ networks.moonriver.api3.anuqrnguint256array }}`       |
-    |      Quintessence QRNG Airnode Address      |      `{{ networks.moonriver.api3.quintessenceairnode }}`       |
-    |       Quintessence QRNG Airnode xpub        |        `{{ networks.moonriver.api3.quintessencexpub }}`        |
-    |     Quintessence Endpoint ID (uint256)      |      `{{ networks.moonriver.api3.quintessenceuint256 }}`       |
-    |    Quintessence Endpoint ID (uint256[])     |    `{{ networks.moonriver.api3.quintessenceuint256array }}`    |
-    |  Quantum Blockchains QRNG Airnode Address   |   `{{ networks.moonriver.api3.quantumblockchainsairnode }}`    |
-    |    Quantum Blockchains QRNG Airnode xpub    |     `{{ networks.moonriver.api3.quantumblockchainsxpub }}`     |
-    |  Quantum Blockchains Endpoint ID (uint256)  |   `{{ networks.moonriver.api3.quantumblockchainsuint256 }}`    |
-    | Quantum Blockchains Endpoint ID (uint256[]) | `{{ networks.moonriver.api3.quantumblockchainsuint256array }}` |
-
-=== "Moonbase Alpha"
-
-    |               Variable               |                         Value                          |
-    |:------------------------------------:|:------------------------------------------------------:|
-    |     TestNet QRNG Airnode Address     |   `{{ networks.moonbase.api3.testnetqrngairnode }}`    |
-    |      TestNet QRNG Airnode xpub       |     `{{ networks.moonbase.api3.testnetqrngxpub }}`     |
-    |  TestNet QRNG Endpoint ID (uint256)  |   `{{ networks.moonbase.api3.testnetqrnguint256 }}`    |
-    | TestNet QRNG Endpoint ID (uint256[]) | `{{ networks.moonbase.api3.testnetqrnguint256array }}` |
-
-For a complete list of all the QRNG Providers, please refer to [API3's documentation](https://docs.api3.org/reference/qrng/providers.html){target=\_blank}.
 
 ## Additional Resources {: #additional-resources }
 
 Here are some additional developer resources:
 
 - [API3 Market](https://market.api3.org/moonbeam){target=\_blank}
 - [API3 Docs](https://docs.api3.org){target=\_blank}
-    - [Getting started with dAPIs](https://docs.api3.org/guides/dapis/){target=\_blank}
-    - [Getting started with QRNG](https://docs.api3.org/guides/qrng/){target=\_blank}
+    - [Getting started with dAPIs](https://airnode-docs.api3.org/reference/airnode/latest/concepts/airnode.html){target=\_blank}
```

Comment:
Let's just remove this link if they don't really have a get started page for dAPIs anymore